src/NodeServices/IAnalyticsClient.cs

```diff
@@ -462,6 +462,21 @@ public enum Actions
         /// Export event, e.g. tracks the ExportSettings event
         /// </summary>
         Export,
+
+        /// <summary>
+        /// Timed event: tracks startup time
+        /// </summary>
+        Startup,
+
+        /// <summary>
+        /// Timed event: tracks view startup time
+        /// </summary>
+        ViewStartup,
```

Comment:
Thanks for the clean up, we need to add these new waypoints to Analytics.NET repo

Reply:
These already exist in the analytics repo.